Formula/fltk.rb

```diff
@@ -1,8 +1,12 @@
 class Fltk < Formula
   desc "Cross-platform C++ GUI toolkit"
   homepage "https://www.fltk.org/"
-  url "https://www.fltk.org/pub/fltk/1.3.6/fltk-1.3.6-source.tar.gz"
-  sha256 "9aac75ef9e9b7bd7b5338a4c0d4dd536e6c22ea7b15ea622aa1d8f1fa30d37ab"
+  license "LGPL-2.0-only" => { with: "FLTK-exception" }
+
+  stable do
+    url "https://www.fltk.org/pub/fltk/1.3.6/fltk-1.3.6-source.tar.gz"
+    sha256 "9aac75ef9e9b7bd7b5338a4c0d4dd536e6c22ea7b15ea622aa1d8f1fa30d37ab"
+  end
```

Comment:
We don't need a `stable` block when it's just a URL and checksum. Put this back, please.

Reply:
Done, now back to original placement. May be worth adding this check to `brew audit` too.